erpnext/accounts/doctype/purchase_invoice/purchase_invoice.js

```diff
@@ -132,6 +144,25 @@ erpnext.accounts.PurchaseInvoice = erpnext.buying.BuyingController.extend({
 			});
 		}
 	},
+	validate_inter_company_invoice: function(frm) {
+		var me = this;
+		frappe.call({
+			method: "erpnext.accounts.doctype.sales_invoice.sales_invoice.validate_inter_company_invoice",
```

Comment:
call this from inside `erpnext.accounts.doctype.purchase_invoice.purchase_invoice.make_inter_company_sales_invoice `